### PR TITLE
Extensions: Zoninator - Improve loading experience on 'Edit Zone' view

### PR DESCRIPTION
**Pull request by ice9js**
This PR aims to improve the loading experience on *Edit Zone* view of the extension. The two improvements are:

- Save buttons are disabled when the data is still loading.
- Placeholders are shown when zone's posts are still loading.

screen shot 2017-10-05 at 17 33 30

# Testing

- Go to `/extensions/zoninator` and select a zone.
- Pay attention to the disabled save buttons and post placeholders in the *Zone content* section while the data is being loaded.

### CODE REVIEW

**Comment by rickybanister** [comment]
I'm blanking a little bit on what this looks like post-loading, but I'd suggest we go with 'compact cards' instead of what we have here. It looks a little off to have that 1px space between the inner cards. Compact cards will automatically render with a single border between the two items. We could also use 'clickable cards' for the zones so that you automatically get the 'link' for selection. 

But I'm just making those recommendations based on the screenshot.

**Comment by ice9js** [comment]
@rickybanister Here's a screenshot of the same part with everything loaded:

screen shot 2017-10-05 at 22 11 28

This one also features spaces between the cards and it seems to be an effect of `SortableList` - the list supports drag & drop. That's why I chose to imitate this with the placeholders.

**Comment by rickybanister** [comment]
Oh, I didn't realize it was sortable. Should there be some visual indicator that it can be sorted?

File: client/extensions/zoninator/components/forms/zone-content-form/post-placeholder.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import SectionHeader from 'components/section-header';
+
+const PostPlaceholder = () => {
+	const className = 'zoninator__zone-list-item is-placeholder';
```
Comment (donnapep) [inline client/extensions/zoninator/components/forms/zone-content-form/post-placeholder.jsx]: If you named this class `zone-content-form__placeholder` then you wouldn't have any linter issues and you could get rid of the `className` variable, which would make this component even more succinct. To be absolutely certain of no name collisions, the CSS selector would then be `.zoninator__main .zone-content-form__placeholder`.

**Comment by ice9js** [comment]
@rickybanister There's the line just above the posts that says you can sort them. I noticed a similar approach was taken with the share buttons [here](https://wordpress.com/sharing/buttons).  
The only other place where this component is used is in the media modal where the whole area underneath the items has a slightly different background which makes it a little more obvious without the need for explicit instructions. Do you think that would be a better idea ?

**Comment by rickybanister** [comment]
Adding a 'move' icon to the left of the zone titles would be better I think.

**Comment by ice9js** [comment]
I'm merging this for now and I'll create a new PR for just the icon as there's no 'move' Gridicon available right now.

**Pull request merged**